cmd/minikube/cmd/delete.go

```diff
@@ -108,7 +110,8 @@ func deleteContainersAndVolumes(ociBin string) {
 		klog.Infof("error delete containers by label %q (might be okay): %+v", delLabel, errs)
 	}
 
-	errs = oci.DeleteAllVolumesByLabel(ociBin, delLabel)
+	ctx := context.Background()
```

Comment:
nit: how about a name that is more clear this is context for delete ? maybe `delCtx` ?

Reply:
does this one need a timeout too ?

Reply:
thanks, fixed